packages/core/src/view/Graph.ts

```diff
@@ -90,6 +91,14 @@ export const defaultPlugins: GraphPluginConstructor[] = [
 class Graph extends EventSource {
   container: HTMLElement;
 
+  /** HTML canvas instance */
+  htmlCanvas: HtmlCanvas2D;
+
+  /** 
+   * Specifies if the cells should be rendered by HTML canvas.
+   */
+  useCanvas = false;
```

Comment:
IHMO, this could be handled using the DIALECT setting as it was previously done in mxGraph to support VML: https://github.com/jgraph/mxgraph/blob/v4.2.2/javascript/src/js/shape/mxShape.js#L8-L23
AFAIK, the code related to the dialect settings still exist in maxGraph.

In addition, having the HtmlCanvas instance here in the Graph class increase the God object syndrom: all classes are going to use it, and this increases the coupling.